keyboards/handwired/dactyl/matrix.c

```diff
@@ -166,25 +162,25 @@ void init_expander(void) {
 
     /*
     Pin direction and pull-up depends on both the diode direction
-    and on whether the column register is 0 ("A") or 1 ("B"):
+    and on whether the column register is GPIOA or GPIOB
     +-------+---------------+---------------+
     |       | ROW2COL       | COL2ROW       |
     +-------+---------------+---------------+
-    | Reg 0 | input, output | output, input |
+    | GPIOA | input, output | output, input |
     +-------+---------------+---------------+
-    | Reg 1 | output, input | input, output |
+    | GPIOB | output, input | input, output |
     +-------+---------------+---------------+
     */
 
-#if (EXPANDER_COLUMN_REGISTER == 0)
+#if (EXPANDER_COL_REGISTER == GPIOA)
```

Comment:
Geez, I really wrote half of this feature and committed it without any testing, huh?

Reply:
I guess so :) Apart from this little typo, you've done a great job and it's a joy working with an expander thanks to your code! To be honest I haven't done any serious testing with the cols on A and rows on B, but in theory it should work.